src/components/FileMessage.tsx

```diff
@@ -38,14 +40,23 @@ export default function FileMessage(props: Props) {
           root!
         )}
         */}
-      <img
-        className="sendbird-ai-widget-file-message-image"
-        src={message.url}
-        alt={''}
-        onLoad={() => {
-          scrollToBottom();
-        }}
-      />
+      {match(message)
+        .when(isVideoMessage, () => (
+          // eslint-disable-next-line jsx-a11y/media-has-caption
+          <video controls className="sendbird-ai-widget-file-message">
+            <source src={message.url} type={message.type} />
+          </video>
+        ))
+        .otherwise(() => (
+          <img
+            className="sendbird-ai-widget-file-message"
+            src={message.url}
+            alt={''}
+            onLoad={() => {
+              scrollToBottom();
+            }}
+          />
+        ))}
```

Comment:
Please avoid using `ts-pattern` as we will need to drop several libraries in the near future.

Reply:
ah ok will do! I did not know this.

Reply:
no problem! One of the most important things for a widget is to minimize its size as much as possible.